### PR TITLE
TW-2144 Update search result list item

### PR DESCRIPTION
**Pull request by KhaledNjim**
## Ticket
- #2144 

## Impact description
Update search list item
## Pre Merge
- https://github.com/linagora/linagora-design-flutter/pull/54
## Resolved
**Attach screenshots or videos demonstrating the changes**
- Web:
image

image

image

image

- Android:
- IOS:

### CODE REVIEW

File: lib/pages/chat_search/chat_search_view.dart [hunk 1]
```diff
@@ -196,74 +201,62 @@ class _SearchItem extends StatelessWidget {
...
       builder: (context, snapshot) {
         final user = snapshot.data ?? event.senderFromMemoryOrFallback;
-        return Padding(
+        return Container(
           padding: ChatSearchStyle.itemMargin,
-          child: InkWell(
-            hoverColor: LinagoraRefColors.material().primary[99],
-            borderRadius:
-                BorderRadius.circular(ChatSearchStyle.itemBorderRadius),
+          height: ChatSearchStyle.itemHeight,
```
Comment (nqhhdev) [inline lib/pages/chat_search/chat_search_view.dart:206]: Why not apply style from system?

Comment (KhaledNjim) [reply lib/pages/chat_search/chat_search_view.dart:206]: Its applied, check the child of this container

Comment (nqhhdev) [reply lib/pages/chat_search/chat_search_view.dart:206]: IMO: 
- `TwakeListItem` included `height` no need to wrap height outside.
- You can move padding into `TwakeListItem` as well

Screenshot 2024-12-16 at 13 00 52

Comment (KhaledNjim) [reply lib/pages/chat_search/chat_search_view.dart]: Updated

Comment (nqhhdev) [reply lib/pages/chat_search/chat_search_view.dart]: Pls check my comment in your PR

https://github.com/linagora/linagora-design-flutter/pull/54/files#r1887803563

**Review by hoangdat — COMMENTED** [review]
look good. but need to fix in https://github.com/linagora/linagora-design-flutter/pull/54

**Comment by hoangdat** [comment]
Merged https://github.com/linagora/linagora-design-flutter/pull/54:  `90f11f5350c691a6244dfbfadfbf53795acf3151`. Please rebase

**Review by hoangdat — APPROVED** [review]

**Pull request merged**